tests/contest/contest/src/main.rs

```diff
@@ -107,6 +108,7 @@ fn main() -> Result<()> {
     let intel_rdt = get_intel_rdt_test();
     let sysctl = get_sysctl_test();
     let scheduler = get_scheduler_test();
+    let io_p_test = get_io_priority_test();
```

Comment:
I don't think it is worth to omit the name.
```suggestion
    let io_priority_test = get_io_priority_test();
```